CHANGELOG.md

```diff
@@ -2,11 +2,16 @@
 
 ## v1.0.0-beta3 (unreleased)
 
+* Add `task.MustRegister` convenience function which fails fast by panic-ing
```

Comment:
not sure if "panic-ing" is right here. panicing? spell checker tells me that's wrong. panicking?